packages/venia-ui/lib/components/CartPage/PriceSummary/discountSummary.js

```diff
@@ -40,17 +41,22 @@ const getDiscount = (discounts = []) => {
 const DiscountSummary = props => {
     const classes = useStyle({}, props.classes);
     const discount = getDiscount(props.data);
+    const { formatMessage } = useIntl();
+
+    const label = discount.label
+        ? discount.label
+        : formatMessage({
+              id: 'discountSummary.lineItemLabel',
+              defaultMessage: 'Discounts applied'
+          });
```

Comment:
I don't think this was part of the scope of this ticket? Was there a reason we want to replace the default message with the source label?

Reply:
It was part of the acceptance criterias to have the source label :  https://jira.corp.magento.com/browse/PWA-2227